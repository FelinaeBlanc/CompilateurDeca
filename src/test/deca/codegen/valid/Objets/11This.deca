class A {
    int x = -4 ;
    int y = 3;
    int z = 2;

    float a = 0+5;
    float b = 13.02;
    float c = this.a ;
}

{
    A a = new A() ;
    println(a.x);
    println(a.y + a.x);
    println(a.c);

}
