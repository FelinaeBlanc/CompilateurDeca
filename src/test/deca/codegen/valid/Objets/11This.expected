-4
-1
5.00000e+00
